content/en/docs/09/_index.md

```diff
@@ -25,37 +21,37 @@ CONTAINER ID        IMAGE               COMMAND                  CREATED
 6f08ac657320        mariadb             "docker-entrypoint..."   5 hours ago         Up 3 hours          3306/tcp             mariadb-container
 ```
 
-Sadly, before we can link the frontend and backend we have to get rid of the existing containers.
+We will link the containers in a new network. But first get rid of the currently running ones:
 
 ```bash
 docker stop apache-php mariadb-container mariadb-container-with-existing-external-volume
 docker rm apache-php mariadb-container mariadb-container-with-existing-external-volume
 ```
 
-To enable the communication between two or more Docker containers you have to use Docker network. Per default there are three networks available:
+To enable communication between Docker containers, use `docker network`. Per default Docker has three networks available. Verify that:
 
 ```bash
 docker network ls
 ```
-
+Will output the following:
 ```
 NETWORK ID          NAME                DRIVER              SCOPE
 9233283df4a6        bridge              bridge              local
 640877f8aec4        host                host                local
 72f9a9996909        none                null                local
 ```
 
-For this exercise we are creating our own network with:
+For this exercise create a new network with:
 
 ```bash
 docker network create container-basics-training
 ```
 
-If you now rerun the list command for Docker networks you should see the newly created network.
+If you now rerun the list command for Docker networks `container-basics-training` will show up.
```

Comment:
```suggestion
If you now rerun the list command for Docker networks, `container-basics-training` will now show up.
```

Reply:
To times now is too much imho. changed it to:
If you rerun the list command for Docker networks now, `container-basics-training` will now up.